ext/dcos-installer/dcos_installer/action_lib/__init__.py

```diff
@@ -322,6 +323,11 @@ def _add_prereqs_script(chain):
   exit 0
 fi
 
+if [ -f /opt/prereqs.installed ]; then
```

Comment:
Can we add `dcos` to the name of this somehow / somewhere? (`/opt/dcos-prereqs.installed`)